### PR TITLE
Add spearman job

### PR DESCRIPTION
**Pull request by oyvindeide**

### CODE REVIEW

**Comment by jondequinor** [comment]
Could you describe what this does? Preferably in the code.

File: semeio/jobs/scripts/spearman_correlation.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        cophenetic distance than `t`.
+        """,
+    )
+    parser.add_argument(
+        "--output-file",
+        required=False,
+        type=str,
+        help="Name of the outputfile. The format will be yaml.",
+    )
+    parser.add_argument(
```
Comment (jondequinor) [inline semeio/jobs/scripts/spearman_correlation.py]: Why would you want a dry run?

File: semeio/jobs/spearman_correlation_job/job.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    measured_data.remove_inactive_observations()
+
+    simulated_data = measured_data.get_simulated_data()
+
+    correlation_matrix = _calculate_correlation_matrix(simulated_data)
+
+    clusters = _cluster_analysis(correlation_matrix, threshold)
+
+    columns = correlation_matrix.columns
+    data = list(
```
Comment (jondequinor) [inline semeio/jobs/spearman_correlation_job/job.py]: Perhaps a small comment as to why this is done?

File: tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import os
+import pytest
+
+@pytest.fixture()
+def setup_tmpdir(tmpdir):
+    cwd = os.getcwd()
+    tmpdir.chdir()
+    yield
+    os.chdir(cwd)
```
Comment (jondequinor) [inline tests/conftest.py]: There's a newline missing here.

File: semeio/jobs/scripts/spearman_correlation.py (hunk 1 above)
Comment (oyvindeide) [reply semeio/jobs/scripts/spearman_correlation.py]: If the user would like to know what clusters are created, but would not necessarily run the scaling. But doesn't make much sense without more printing i suppose.

Comment (oyvindeide) [reply semeio/jobs/scripts/spearman_correlation.py]: Printing added

**Pull request merged**